src/awkward/_v2/_util.py

```diff
@@ -627,594 +636,536 @@ def wrap(content, behavior=None, highlevel=True, like=None):
-#                     if isinstance(x, ak._v2.contents.RegularArray):
-#                         if maxsize > 1 and x.size == 1:
-#                             tmpindex = ak._v2.contents.Index64(
-#                                 nplike.repeat(
-#                                     nplike.arange(len(x), dtype=np.int64), maxsize
-#                                 )
-#                             )
-#                 nextinputs = []
-#                 for x in inputs:
-#                     if isinstance(x, ak._v2.contents.RegularArray):
-#                         if maxsize > 1 and x.size == 1:
-#                             nextinputs.append(
-#                                 ak._v2.contents.IndexedArray64(
-#                                     tmpindex, x.content[: len(x) * x.size]
-#                                 ).project()
-#                             )
-#                         elif x.size == maxsize:
-#                             nextinputs.append(x.content[: len(x) * x.size])
-#                         else:
-#                             raise ValueError(
-#                                 "cannot broadcast RegularArray of size "
-#                                 "{0} with RegularArray of size {1}".format(
-#                                     x.size, maxsize
-#                                 )
-#
-#                             )
-#                     else:
-#                         nextinputs.append(x)
-
-#                 maxlen = max(
-#                     [len(x) for x in nextinputs if isinstance(x, ak._v2.contents.Content)]
-#                 )
-#                 outcontent = apply(nextinputs, depth + 1, user)
-#                 assert isinstance(outcontent, tuple)
-
-#                 return tuple(
-#                     ak._v2.contents.RegularArray(x, maxsize, maxlen) for x in outcontent
-#                 )
-
-#             elif not all_same_offsets(nplike, inputs):
-#                 fcns = [
-#                     custom_broadcast(x, behavior)
-#                     if isinstance(x, ak._v2.contents.Content)
-#                     else None
-#                     for x in inputs
-#                 ]
-
-#                 first, secondround = None, False
-#                 for x, fcn in zip(inputs, fcns):
-#                     if (
-#                         isinstance(x, listtypes)
-#                         and not isinstance(x, ak._v2.contents.RegularArray)
-#                         and fcn is None
-#                     ):
-#                         first = x
-#                         break
-
-#                 if first is None:
-#                     secondround = True
-#                     for x in inputs:
-#                         if isinstance(x, listtypes) and not isinstance(
-#                             x, ak._v2.contents.RegularArray
-#                         ):
-#                             first = x
-#                             break
+def broadcast_and_apply(  # noqa: C901
```

Comment:
`broadcast_and_apply` is in `_broadcasting.py` now. That's the one that has a "depth_context" and "nested_context" now. I'm sorry—I should have removed the commented out version from this file.